bin/serve.sh

```diff
@@ -1,23 +1,9 @@
 #!/usr/bin/env sh
 set -e
 
-dockerize -wait tcp://postgres:5432 -timeout 1m
```

Comment:
since dockerize has been removed, should we still be installing it?

Reply:
It's not removed. We install it via `wget`.